src/startup.lisp

```diff
@@ -42,7 +35,17 @@
   (setq *production* t)  ;; used by EMOTIQ:PRODUCTION-P in Crypto
   (message-running-state "from command line")
   (actors:install-actor-system)
-  (main))
+  (main)
+  (emotiq/sim:initialize)
+  (emotiq/sim:run :cloaked nil))
+
+;; "Entry Point" for development - does nothing, just load and go
```

Comment:
This comment is incorrect:  it isn't true that this function does nothing.  In fact it is meant to be the main loop of the application.  

Better to do as a docstring in any event.